lib/ansible/modules/remote_management/redfish/idrac_redfish_config.py

```diff
@@ -14,7 +14,7 @@
 DOCUMENTATION = '''
 ---
 module: idrac_redfish_config
-version_added: '2.8'
+version_added: '2.10'
```

Comment:
This should be 2.8 (the version when this module was originally added)

Reply:
Thanks for your continued guidance. Fixed.